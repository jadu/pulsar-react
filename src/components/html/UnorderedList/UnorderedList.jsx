import React from 'react';

export default class UnorderedList extends React.Component {

  render() {
    let {
      children,
      ...props
    } = this.props;

    return (
      <ul {...props}>
        {children}
      </ul>
    );
  }
}