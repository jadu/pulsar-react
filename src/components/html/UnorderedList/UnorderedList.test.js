'use strict';

import React from 'react';
import { render } from 'enzyme';

import UnorderedList from './UnorderedList';

it('renders the base elements', () => {
  const tree = render(
    <UnorderedList>foo</UnorderedList>
  );
  expect(tree).toMatchSnapshot();
});