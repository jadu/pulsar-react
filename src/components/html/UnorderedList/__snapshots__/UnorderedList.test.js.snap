// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the base elements 1`] = `
<ul>
  foo
</ul>
`;
